libs/ui-lib-tests/cypress/integration/create-multinode/0-create-cluster.cy.ts

```diff
@@ -28,10 +28,13 @@ describe(`Assisted Installer Multinode Cluster Installation`, () => {
 
       commonActions.getInfoAlert().should('not.exist');
       commonActions.toNextStepAfter('Cluster details');
-      cy.wait('@create-cluster');
+
+      cy.wait('@create-cluster').then(({ request }) => {
+        expect(Object.keys(request.body)).not.to.contain('platform');
+      });
```

Comment:
Following [this discussion](https://redhat-internal.slack.com/archives/C05KWUC1E2X/p1690974339940609), we should assert that if the platform is set and the type is 'baremetal' then user_managed_networking = false, maybe this is the right place to do it.

Reply:
This is definitely not the right place to do that. (`with-user-managed-networking.cy.ts` might be better)

But following that discussion it isn't 100% that we will be setting baremetal+UMN from the cluster details. So let's not rush this.